added an hierarchical toc based on architecture for linux installation on ODBC

dwicinas opened the pull request:
## What Changed?

Please list, at a high level, what changed in your branch. If you changed content, include the product, section, and version as applicable. **Please remove the user instructions including the examples before merging the PR.**

**Examples**

- Fixed typo in EPAS 13 epas_inst_linux guide
- Added more detail to ARK 3.5 getting started guide introduction
- Fixed broken link on `/supported-open-source/pgbackrest/05-retention_policy/` page

## Checklist

Please check all boxes that apply (`[ ]` is unchecked, `[x]` is checked)

**Content**

- [ ] This PR adds new content
- [ ] This PR changes existing content
- [ ] This PR removes existing content
- [ ] This PR is for a release, please add this tag:

drothery-edb reviewed (commented):
This one needs a little more work. Need to change to sentence case in the install topics and there is at least one broken link - the link to x86 RH 7 on the top level index. I didn't check all the links.

dwicinas commented:
@drothery-edb I fixed that broken link and changed to sentence case for the upgrading topic. I didn't see other sentence case issues. Did you see others?
